validator/Cargo.toml

```diff
@@ -10,6 +10,7 @@ publish = false
 anyhow = "1"
 bytes = "1"
 kubert = { version = "0.19.0", features = ["log"] }
+linkerd-reinitialize-pods = { path = "../reinitialize-pods" }
```

Comment:
This means that the validator will now take on all of the dependencies of the controller, which isn't desirable if we're doing this to reuse a constant. It would be better to leave the validator unchanged.